fab/operations/supervisor.py

```diff
@@ -72,14 +73,41 @@ def set_celery_supervisorconf():
         'logistics_background_queue':   ['supervisor_celery_logistics_background_queue.conf'],
         'async_restore_queue':          ['supervisor_celery_async_restore_queue.conf'],
         'flower':                       ['supervisor_celery_flower.conf'],
-        }
+    }
 
     queues = _get_celery_queues()
+    if 'periodic' in queues and env.host != queues['periodic'].get('server_whitelist'):
+        show_periodic_server_whitelist_message_and_abort(env)
     for queue, params in queues.items():
         for config_file in conf_files[queue]:
             _rebuild_supervisor_conf_file('make_supervisor_conf', config_file, {'celery_params': params})
 
 
+def show_periodic_server_whitelist_message_and_abort(env):
+    utils.abort(
+        "\n\n"
+        "You're seeing this message as part of an effort to reduce the chance of\n"
+        "deploying celery beat, which sends SMSes, fires off reminders,\n"
+        "and triggers forwarders, among other things, unintentionally.\n\n"
+        "For example, during a migration, you initially want to deploy everything\n"
+        "except for celery beat, because you don't want both the old and new servers\n"
+        "running celery beat; that screws up queuing, etc.\n\n"
```

Comment:
Oh? I thought this _was_ the problem there—that celery beat was running on the new (virginia) setup while the erstwhile production setups was still running. I wasn't aware or forgot that SMS reminders are now triggered by a `run_reminder_queue` management command that replaces the polling that used to be done through celery beat. That's an analogous thing, so we want basically the same error to show up there.

That may mean slightly changing where this check lives, etc. Can you think of a full list of "things"/processes/services that should not get deployed to a pre-production setup that's currently in "mirror" mode (the way virginia was before we shut off chicago and flipped the DNS)? So far I'm thinking:
- celery beat
- run_reminder_queue
- sms queue (though this may be redundant if nothing's creating these tasks)
- reminder queue (ditto)
- pillow retry queue (not sure how these get triggered)

Anything else? Another thought to reiterate is I think it's more important to disable the thing putting things _on_ a celery queue than it is to disable the things that _process_ the queues. If you only disabled processing the celery queue then when you do the flip, there will be all these (duplicate) things in the queue that have already been processed by the previous production.